common/lib/capa/capa/safe_exec/safe_exec.py

```diff
@@ -146,6 +146,7 @@ def safe_exec(
             python_path=python_path, extra_files=extra_files, slug=slug,
         )
     except SafeExecException as e:
+        exception = e
```

Comment:
BOM-607

Reply:
Won't this just raise the same error but for `exception` instead of `e`?

eg. `UnboundLocalError: local variable 'exception' referenced before assignment`

I think you'll need to define exception outside of the except clause so that it doesn't throw this error.

Reply:
In py2 all the variables created in exception block as well as exception variable are available outside block. but in py3 exception variable is not available outside the block but other created variables are available outside the block.

Reply:
```
try:
    raise ValueError('It Works')
except Exception as e:
    test = 'test'
print(test)
print(e)
```
If you run the following code in py2 and py3, it will help you understand what i said earlier.